lib/command/config.rb

```diff
@@ -21,11 +21,6 @@ class Config < Base
     EX
 
     def call # rubocop:disable Metrics/MethodLength
-      if config.org_comes_from_env
```

Comment:
this file needs to change so that `load_app_config` is called lazily

Reply:
Let's handle this in a separate PR.